src/main/webapp/match-types.tsx

```diff
@@ -0,0 +1,58 @@
+import * as React from 'react'
+import FacetedDropdown from './faceted-dropdown'
+import gql from 'graphql-tag'
+import { useQuery } from '@apollo/react-hooks'
+import LinearProgress from '@material-ui/core/LinearProgress'
+import { getIn } from 'immutable'
+const { useApolloFallback } = require('./react-hooks')
+const MATCHTYPE_ATTRIBUTE = gql`
+  query getAttributeSuggestionList {
+    systemProperties {
+      basicSearchMatchType
+    }
+  }
+`
+
+type Props = {
+  value: string[]
+  errors: any
+  onChange: (val: string[]) => void
+  matchTypeAttribute?: string
+}
+
+const MatchTypes = (props: Props) => {
+  const errors = props.errors
+
+  const facetAttribute = props.matchTypeAttribute || 'datatype'
```

Comment:
I need to update basic search's to/fromfilterTree implementations to allow for a configurable match type attribute